ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Examples.hs

```diff
@@ -245,7 +251,7 @@ examples value txBody auxiliaryData = Golden.Examples {
     results = labelled [
           ("LedgerTip",              SomeResult GetLedgerTip (blockPoint blk))
         , ("EpochNo",                SomeResult GetEpochNo 10)
-        , ("EmptyPParams",           SomeResult GetCurrentPParams SL.emptyPParams)
+        , ("EmptyPParams",           SomeResult GetCurrentPParams def)
```

Comment:
Just wondering: would it make sense to make it a `Monoid` so we can use `mempty` instead of `def`? `Monoid` has laws :slightly_smiling_face:

Reply:
It might be reasonable for `PParams` to be a `Semigroup`, but a monoid? No. Precisely because the default instance would _not_ satisfy any monoid laws - the default values are default values, not a monoidal unit.

Reply:
(`PParams' Maybe` _could_ be a monoid, but its unit would not be the thing we want here as a default)